tests/mask.py

```diff
@@ -138,23 +135,28 @@ def testOrMasks(self):
         self.mask2 |= self.mask1
         self.mask1 |= self.val2
         
-        self.assertEqual(self.mask1.get(0, 0), self.val1 | self.val2)
-        self.assertEqual(self.mask2.get(0, 0), self.val1 | self.val2)
-    
+        self.assertMasksEqual(self.mask1, self.mask2)
+        expect = np.zeros_like(self.mask1.getArray())
+        expect |= self.val2 | self.val1
```

Comment:
At risk of being too picky, I think the following would be slightly easier to follow because the user doesn't have to know the initial value (which is zero, because I looked, but still...)
expect[:] = self.val2 | self.val1